devtools/conda-envs/mers-docking2.yaml

```diff
@@ -0,0 +1,23 @@
+name: mers-docking2
```

Comment:
There are many conda .yamls currently, I think it would make more sense to remove outdated ones and include a quick readme that describes which is which.

Reply:
yes, I think we should do that in a future pr: https://github.com/choderalab/covid-moonshot-ml/issues/41